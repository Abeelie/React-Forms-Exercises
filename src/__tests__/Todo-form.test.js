import React from "react";
import { render } from "@testing-library/react";
import TodoForm from "../Components/todo-form/Todo-form";



it("renders without crashing", function () {
  render(<TodoForm />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<TodoForm />);
  expect(asFragment()).toMatchSnapshot();
});