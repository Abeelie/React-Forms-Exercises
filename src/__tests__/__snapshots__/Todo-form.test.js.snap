// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="todo-form"
  >
    <label
      for="name"
    >
      Todo:
    </label>
    <input
      id="name"
      name="name"
      value=""
    />
    <button
      id="btn"
    >
      Add todo
    </button>
  </form>
</DocumentFragment>
`;
